DockingStation docking_station
---
# result definition
bool success
string message